lib/game.rb

```diff
@@ -0,0 +1,61 @@
+class Game
+  def initialize(_player1, _player2)
+    @n = 0
+    @moves_used = []
+  end
+
+  def start
+    system 'clear'
+
+    puts "
+      \t ██╗░░░░░░█████╗░░█████╗░██████╗░██╗███╗░░██╗░██████╗░░░░░░░░░░
+      \t ██║░░░░░██╔══██╗██╔══██╗██╔══██╗██║████╗░██║██╔════╝░░░░░░░░░░
+      \t ██║░░░░░██║░░██║███████║██║░░██║██║██╔██╗██║██║░░██╗░░░░░░░░░░
+      \t ██║░░░░░██║░░██║██╔══██║██║░░██║██║██║╚████║██║░░╚██╗░░░░░░░░░
+      \t ███████╗╚█████╔╝██║░░██║██████╔╝██║██║░╚███║╚██████╔╝██╗██╗██╗
+      \t ╚══════╝░╚════╝░╚═╝░░╚═╝╚═════╝░╚═╝╚═╝░░╚══╝░╚═════╝░╚═╝╚═╝╚═╝"
+
+    # Create Variable to Tic tac toe's board
+    @game_board = [[1, 2, 3], [4, 5, 6], [7, 8, 9]]
+  end
+
+  # Display the Tic tac toe's board
+  def display_board
+    (0..2).each do |i|
+      puts "\t +---+---+---+"
+      print "\t"
+      (0..7).each do |j|
+        print ' | ' unless j.odd?
+        print @game_board[i][((j * 3) / 7).to_i] unless j.even?
+      end
+      puts i == 2 ? "\n\t +---+---+---+" : "\t nil"
```

Comment:
```suggestion
      puts i == 2 ? "\n\t +---+---+---+" : "\t"
```
I thought there is an undefined variable that printing out `nil` :smile: